src/main/java/de/zalando/aruha/nakadi/service/StrategiesRegistry.java

```diff
@@ -9,20 +9,13 @@
 public class StrategiesRegistry {
 
     public static final PartitionStrategyDescriptor HASH_PARTITION_STRATEGY =
-            new PartitionStrategyDescriptor("hash", "This strategy will use the event " +
```

Comment:
We had this discussion since I once proposed to remove the description from the API. I would strongly vote for not removing the description from the code (and rather remove it from the API or keep it in both places).

I see the strategies not as part of the api but something that theoretically could be configured and is a open registry. A description for a strategy hence should be come a long with the strategy and not with Nakadi's API. That's my 2c.

What I meant, when we discussed this offline was that I would change the field in the API from

```
partition_strategy:
        allOf:
          - $ref: '#/definitions/EventPartitionStrategy'
```

to

```
partition_strategy:
        type: string
```

referring to the name of a strategy.